Tests/Functional/FunctionalTestCate.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SyliusConsumerBundle\Tests\Functional;
+
+use Sulu\Bundle\SyliusConsumerBundle\Tests\Service\GatewayClient;
+use Sulu\Bundle\TestBundle\Testing\SuluTestCase;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+class FunctionalTestCate extends SuluTestCase
```

Comment:
should be something like: `AbstractGatewayClientTestCase`

Reply:
@alexander-schranz but there isnt any abstract method in it. So i think the `GatewayClientTestCase` would make more sense.